### PR TITLE
Added SaferNet workflow

### PR DESCRIPTION
**Pull request by stephenhand**
New prod workflow for SaferNet

### CODE REVIEW

File: .github/workflows/safernet_production.yml [hunk 1]
```diff
@@ -0,0 +1,127 @@
+# This is a basic workflow to publish a Twilio function with Github Actions
+
+name: SaferNet Brazil Production release
```
Comment (nickhurlburt) [inline .github/workflows/safernet_production.yml:3]: the staging one is 'SaferNet Staging release' without the Brazil... maybe we should do 'SaferNet Production release' to keep consistent with that

**Review by GPaoloni — APPROVED** [review]
Looks good!

File: .github/workflows/safernet_production.yml [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+          env_variable_name: "FLEX_PROXY_SERVICE_SID"
+      - name: Set HRM static key to make HRM calls
+        uses: "marvinpinto/action-inject-ssm-secrets@latest"
+        with:
+          ssm_parameter: "PROD_TWILIO_SNBR_HRM_STATIC_KEY"
+          env_variable_name: "HRM_STATIC_KEY"
+      - name: Set Survey Workflow SID
+        uses: "marvinpinto/action-inject-ssm-secrets@latest"
+        with:
+          ssm_parameter: "PROD_TWILIO_SNBR_SURVEY_WORKFLOW_SID"
```
Comment (nickhurlburt) [inline .github/workflows/safernet_production.yml:80]: Is `SNBR` consistent with how we've been naming things? (If so, never mind me)

**Review by nickhurlburt — APPROVED** [review]
Rely on @GPaoloni's review more than mine. I just had some naming questions. Otherwise looks good.

**Pull request merged**